platform/jobs/RetirementJobs.groovy

```diff
@@ -174,6 +174,7 @@ job('user-retirement-driver') {
             name('user-retirement-driver')
             nature('shell')
             command(readFileFromWorkspace('platform/resources/user-retirement-driver.sh'))
+            pythonName('PYTHON_3.5')
```

Comment:
Tubular is tested under Python 3.6 as well as Python 3.5:
https://github.com/edx/tubular/blob/410cb3f11b05386ab14a675bc851cfdb08faab7b/tox.ini#L2
Maybe instead use `System-CPython-3.6`?

Or update tubular to test using Python 3.7 - and use `PYTHON_3.7`?